diff -Naur a/rocprim/CMakeLists.txt b/rocprim/CMakeLists.txt
--- a/rocprim/CMakeLists.txt	2019-06-08 21:12:42.155827776 +0200
+++ b/rocprim/CMakeLists.txt	2019-06-08 21:31:12.724823539 +0200
@@ -63,7 +63,8 @@
   DIRECTORY
     "include/"
     "${PROJECT_BINARY_DIR}/rocprim/include/"
-  DESTINATION rocprim/include/
+#  DESTINATION rocprim/include/
+  DESTINATION include/
   FILES_MATCHING
   PATTERN "*.h"
   PATTERN "*.hpp"
@@ -73,9 +74,10 @@
 # Export targets
 rocm_export_targets(
   TARGETS rocprim-targets
-  PREFIX rocprim
+#  PREFIX rocprim
+#  PREFIX
   NAMESPACE roc::
 )
 
 # Create symlinks
-rocm_install_symlink_subdir(rocprim)
+#rocm_install_symlink_subdir(rocprim)
