--- a/cmake/VerifyCompiler.cmake	2019-07-01 16:27:51.524649461 -0700
+++ b/cmake/VerifyCompiler.cmake	2019-07-01 16:27:06.747980569 -0700
@@ -38,7 +38,7 @@
         list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hcc /opt/rocm/hip)
         # Ignore hcc warning: argument unused during compilation: '-isystem /opt/rocm/hip/include'
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-command-line-argument")
-        find_package(hcc REQUIRED CONFIG PATHS /opt/rocm)
+#find_package(hcc REQUIRED CONFIG PATHS /opt/rocm)
         find_package(hip REQUIRED CONFIG PATHS /opt/rocm)
     endif()
 else()
