diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8133253f..5ce05615 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -197,7 +197,7 @@ else()
     set(MIOPEN_USE_MIOPENGEMM 0)
 endif()
 
-option(Boost_USE_STATIC_LIBS "Use boost static libraries" ON)
+option(Boost_USE_STATIC_LIBS "Use boost static libraries" OFF)
 set(BOOST_COMPONENTS filesystem system)
 add_definitions(-DBOOST_ALL_NO_LIB=1)
 find_package(Boost REQUIRED COMPONENTS ${BOOST_COMPONENTS})
