From 40c3b3e9d0e04016e61740a2a609852a7737b5f3 Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Mon, 26 Aug 2019 17:59:12 +0100
Subject: [PATCH] Remove as.h CMakeLists.txt so it doesn't get installed, or
 fails.

---
 ockl/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ockl/CMakeLists.txt b/ockl/CMakeLists.txt
index 8931ed6..ec06e7c 100644
--- a/ockl/CMakeLists.txt
+++ b/ockl/CMakeLists.txt
@@ -21,7 +21,6 @@ install(FILES
         inc/amd_hsa_kernel_code.h
         inc/amd_hsa_queue.h
         inc/amd_hsa_signal.h
-        inc/as.h
         inc/device_amd_hsa.h
         inc/hsa.h
         inc/ockl_hsa.h
-- 
2.23.0

