--- a/amd/dkms/Makefile.orig	2020-05-19 11:34:20.000000000 -0700
+++ b/amd/dkms/Makefile	2020-06-03 00:32:42.591572243 -0700
@@ -194,6 +194,7 @@ export CONFIG_DRM_AMD_DC_DCN1_0=y
 export CONFIG_DRM_AMD_DC_DCN1_01=y
 export CONFIG_DRM_AMD_DC_DCN2_0=y
 export CONFIG_DRM_AMD_DC_DCN2_1=y
+export CONFIG_MMU_NOTIFIER=y
 
 subdir-ccflags-y += -DCONFIG_HSA_AMD
 subdir-ccflags-y += -DCONFIG_DRM_TTM_DMA_PAGE_POOL
@@ -205,5 +206,6 @@ subdir-ccflags-y += -DCONFIG_DRM_AMD_DC_
 subdir-ccflags-y += -DCONFIG_DRM_AMD_DC_DCN1_01
 subdir-ccflags-y += -DCONFIG_DRM_AMD_DC_DCN2_0
 subdir-ccflags-y += -DCONFIG_DRM_AMD_DC_DCN2_1
+subdir-ccflags-y += -DCONFIG_MMU_NOTIFIER
 
 obj-m += scheduler/ amd/amdgpu/ ttm/ amd/amdkcl/
