--- a/rocminfo.cc
+++ b/rocminfo.cc
@@ -1048,7 +1048,6 @@
   // "render")
   // @note: User who are not members of "video"
   // group cannot access DRM services
-  char u_name[32];
   bool member = false;
   struct passwd *pw;
   int num_groups = 0;
@@ -1088,23 +1087,17 @@
     return;
   }
 
-  if (getlogin_r(u_name, 32)) {
-    printf("%sFailed to get user name to check for"
+  pw = getpwuid(getuid());
+  if (pw == NULL || pw->pw_name == nullptr) {
+    printf("%sFailed to get user pwd entry to check for"
            " %s group membership%s\n", COL_RED, kfd_gr_name,
            COL_RESET);
     return;
   }
 
-  pw = getpwnam(u_name); // NOLINT
-  if (pw == NULL) {
-    printf("%sFailed to find pwd entry for user %s%s\n",
-                                                  COL_RED, u_name, COL_RESET);
-    return;
-  }
-
-  (void)getgrouplist(u_name, pw->pw_gid, NULL, &num_groups);
+  (void)getgrouplist(pw->pw_name, pw->pw_gid, NULL, &num_groups);
   groups = new gid_t[num_groups];
-  if (getgrouplist(u_name, pw->pw_gid, groups, &num_groups) == -1) {
+  if (getgrouplist(pw->pw_name, pw->pw_gid, groups, &num_groups) == -1) {
     printf("%sFailed to get user group list%s\n", COL_RED, COL_RESET);
     delete []groups;
     return;
@@ -1112,7 +1105,7 @@
 
   for (int i = 0; i < num_groups; ++i) {
     if (gr_s->gr_gid == groups[i]) {
-      printf("%s%s is member of %s group%s\n", COL_WHT, u_name, kfd_gr_name, COL_RESET);
+      printf("%s%s is member of %s group%s\n", COL_WHT, pw->pw_name, kfd_gr_name, COL_RESET);
       member = true;
       break;
     }
@@ -1121,7 +1114,7 @@
     printf("%s%s is not member of \"%s\" group, the default DRM access "
      "group. Users must be a member of the \"%s\" group or another"
         " DRM access group in order for ROCm applications to run "
-           "successfully%s.\n", COL_RED, u_name, kfd_gr_name, kfd_gr_name, COL_RESET);
+           "successfully%s.\n", COL_RED, pw->pw_name, kfd_gr_name, kfd_gr_name, COL_RESET);
   }
 
   delete []groups;
