--- a/cmake/virtualenv.cmake	2018-09-16 18:40:48.610063501 +0200
+++ b/cmake/virtualenv.cmake	2018-09-16 18:40:53.852063420 +0200
@@ -1,7 +1,7 @@
-# find_package(PythonInterp)
+find_package(PythonInterp 2.7 REQUIRED)
 # # TODO: Check PYTHON_VERSION_MAJOR
 
-find_program(VIRTUALENV_PYTHON_EXE python)
+find_program(VIRTUALENV_PYTHON_EXE python2.7)
 
 
 set(VIRTUALENV_SOURCE_DIR ${CMAKE_BINARY_DIR}/virtualenv-source CACHE PATH "Path to virtualenv source")
