--- a/Tensile/Common.py	2019-05-04 11:40:45.301855169 +0200
+++ b/Tensile/Common.py	2019-05-04 11:41:21.717855030 +0200
@@ -131,7 +131,7 @@
 globalParameters["LibraryClientPath"] = "4_LibraryClient"         # subdirectory for building example library client
 
 # internal, i.e., gets set during startup
-globalParameters["CurrentISA"] = (0,0,0)
+globalParameters["CurrentISA"] = (9,0,0)
 globalParameters["ROCmAgentEnumeratorPath"] = None      # /opt/rocm/bin/rocm_agent_enumerator
 globalParameters["ROCmSMIPath"] = None                  # /opt/rocm/bin/rocm-smi
 globalParameters["AssemblerPath"] = None                # /opt/rocm/bin/hcc
