diff -uprN bk_rocBLAS-rocm-3.5.0/CMakeLists.txt rocBLAS-rocm-3.5.0/CMakeLists.txt
--- bk_rocBLAS-rocm-3.5.0/CMakeLists.txt	2020-07-21 21:57:58.901879868 +0800
+++ rocBLAS-rocm-3.5.0/CMakeLists.txt	2020-07-21 21:59:38.760610766 +0800
@@ -165,37 +165,18 @@ if( BUILD_WITH_TENSILE )
   set( Tensile_LOGIC "asm_full" CACHE STRING "Tensile to use which logic?")
   set( Tensile_ARCHITECTURE "all" CACHE STRING "Tensile to use which architecture?")
   set( Tensile_CODE_OBJECT_VERSION "V2" CACHE STRING "Tensile code_object_version")
-  set( Tensile_COMPILER "hcc" CACHE STRING "Tensile compiler")
+  set( Tensile_COMPILER "hipcc" CACHE STRING "Tensile compiler")
 
   option( Tensile_MERGE_FILES "Tensile to merge kernels and solutions files?" ON )
   option( Tensile_SHORT_FILENAMES "Tensile to use short file names? Use if compiler complains they're too long." OFF )
   option( Tensile_PRINT_DEBUG "Tensile to print runtime debug info?" OFF )
 
-  set( Tensile_TEST_LOCAL_PATH "" CACHE PATH "Use local Tensile directory instead of fetching a GitHub branch" )
   set_property( CACHE Tensile_ARCHITECTURE PROPERTY STRINGS all gfx803 gfx900 gfx906 gfx908)
   set_property( CACHE Tensile_LOGIC PROPERTY STRINGS asm_full asm_lite asm_miopen hip_lite other )
   set_property( CACHE Tensile_CODE_OBJECT_VERSION PROPERTY STRINGS V2 V3 )
   set_property( CACHE Tensile_COMPILER PROPERTY STRINGS hcc hipcc)
 
-  include(virtualenv)
-  if (Tensile_TEST_LOCAL_PATH)
-    virtualenv_install(${Tensile_TEST_LOCAL_PATH})
-    message (STATUS "using local Tensile from ${Tensile_TEST_LOCAL_PATH}, copied to ${Tensile_ROOT}")
-  else()
-    # Use the virtual-env setup and download package from specified repot:
-    set( tensile_fork "ROCmSoftwarePlatform" CACHE STRING "Tensile fork to use" )
-    file (STRINGS "tensile_tag.txt" read_tensile_tag)
-    set( tensile_tag ${read_tensile_tag} CACHE STRING "Tensile tag to download" )
-    virtualenv_install("git+https://github.com/${tensile_fork}/Tensile.git@${tensile_tag}")
-    message (STATUS "using GIT Tensile fork=${tensile_fork} from branch=${tensile_tag}")
-  endif()
-  message(STATUS "Adding ${VIRTUALENV_HOME_DIR} to CMAKE_PREFIX_PATH")
-  list(APPEND CMAKE_PREFIX_PATH ${VIRTUALENV_HOME_DIR})
-  if (TENSILE_VERSION)
-    find_package(Tensile ${TENSILE_VERSION} EXACT REQUIRED HIP LLVM OpenMP PATHS "${INSTALLED_TENSILE_PATH}")
-  else()
-    find_package(Tensile 4.19.0 EXACT REQUIRED HIP LLVM OpenMP PATHS "${INSTALLED_TENSILE_PATH}")
-  endif()
+  find_package(Tensile REQUIRED HIP LLVM OpenMP)
 endif()
 
 # Find HCC/HIP dependencies
