--- a/CMakeLists.txt	2019-04-26 16:21:47.470700776 +0200
+++ b/CMakeLists.txt	2019-04-26 16:22:18.314700659 +0200
@@ -71,7 +71,7 @@
 endif ()
 
 ## Build tests
-add_subdirectory ( ${TEST_DIR} ${PROJECT_BINARY_DIR}/test )
+#add_subdirectory ( ${TEST_DIR} ${PROJECT_BINARY_DIR}/test )
 
 ## Create symlinks for packaging and install
 add_custom_target ( rocprof-link ALL WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
@@ -100,14 +100,14 @@
           PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
           RENAME rocprof )
 # gfx_metrics.xml metrics.xml
-install ( FILES
-          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/metrics.xml
-          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/gfx_metrics.xml
-          DESTINATION lib )
+#install ( FILES
+#          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/metrics.xml
+#          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/gfx_metrics.xml
+#          DESTINATION lib )
 # libtool.so
-install ( FILES ${PROJECT_BINARY_DIR}/test/libtool.so DESTINATION tool )
-install ( FILES ${PROJECT_BINARY_DIR}/test/ctrl DESTINATION tool
-          PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE )
+#install ( FILES ${PROJECT_BINARY_DIR}/test/libtool.so DESTINATION tool )
+#install ( FILES ${PROJECT_BINARY_DIR}/test/ctrl DESTINATION tool
+#          PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE )
 
 ## Packaging directives
 set ( CPACK_GENERATOR "DEB" "RPM" "TGZ" )
