--- a/CMakeLists.txt	2019-10-13 13:24:56.487085553 +0200
+++ b/CMakeLists.txt	2019-10-13 13:27:38.569088401 +0200
@@ -74,16 +74,16 @@
 add_subdirectory ( ${TEST_DIR} ${PROJECT_BINARY_DIR}/test )
 
 ## Create symlinks for packaging and install
-add_custom_target ( rocprof-link ALL WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
-                    COMMAND ${CMAKE_COMMAND} -E create_symlink ../${ROCPROFILER_NAME}/bin/rpl_run.sh rocprof-link )
-add_custom_target ( inc-link ALL WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
-                    COMMAND ${CMAKE_COMMAND} -E create_symlink ../${ROCPROFILER_NAME}/include inc-link )
-add_custom_target ( so-link ALL WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
-                    COMMAND ${CMAKE_COMMAND} -E create_symlink ../${ROCPROFILER_NAME}/lib/${ROCPROFILER_LIBRARY}.so so-link )
+#add_custom_target ( rocprof-link ALL WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+#                    COMMAND ${CMAKE_COMMAND} -E create_symlink ../${ROCPROFILER_NAME}/bin/rpl_run.sh rocprof-link )
+#add_custom_target ( inc-link ALL WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+#                    COMMAND ${CMAKE_COMMAND} -E create_symlink ../${ROCPROFILER_NAME}/include inc-link )
+#add_custom_target ( so-link ALL WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+#                    COMMAND ${CMAKE_COMMAND} -E create_symlink ../${ROCPROFILER_NAME}/lib/${ROCPROFILER_LIBRARY}.so so-link )
 
-set ( CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/${ROCPROFILER_NAME}" )
+#set ( CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/${ROCPROFILER_NAME}" )
 message ( "---------Install-Dir: ${CMAKE_INSTALL_PREFIX}" )
-install ( TARGETS ${ROCPROFILER_TARGET} LIBRARY DESTINATION lib )
+install ( TARGETS ${ROCPROFILER_TARGET} LIBRARY DESTINATION lib64 )
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/inc/rocprofiler.h DESTINATION include )
 # rpl_run.sh tblextr.py txt2xml.sh
 install ( FILES
@@ -94,20 +94,20 @@
           ${CMAKE_CURRENT_SOURCE_DIR}/bin/sqlitedb.py
           DESTINATION bin
           PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE )
-install ( FILES ${PROJECT_BINARY_DIR}/inc-link DESTINATION ../include RENAME ${ROCPROFILER_NAME} )
-install ( FILES ${PROJECT_BINARY_DIR}/so-link DESTINATION ../lib RENAME ${ROCPROFILER_LIBRARY}.so )
-install ( FILES ${PROJECT_BINARY_DIR}/rocprof-link DESTINATION ../bin
-          PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-          RENAME rocprof )
+#install ( FILES ${PROJECT_BINARY_DIR}/inc-link DESTINATION ../include RENAME ${ROCPROFILER_NAME} )
+#install ( FILES ${PROJECT_BINARY_DIR}/so-link DESTINATION ../lib RENAME ${ROCPROFILER_LIBRARY}.so )
+#install ( FILES ${PROJECT_BINARY_DIR}/rocprof-link DESTINATION ../bin
+#          PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
+#          RENAME rocprof )
 # gfx_metrics.xml metrics.xml
-install ( FILES
-          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/metrics.xml
-          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/gfx_metrics.xml
-          DESTINATION lib )
+#install ( FILES
+#          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/metrics.xml
+#          ${CMAKE_CURRENT_SOURCE_DIR}/test/tool/gfx_metrics.xml
+#          DESTINATION lib )
 # libtool.so
-install ( FILES ${PROJECT_BINARY_DIR}/test/libtool.so DESTINATION tool )
-install ( FILES ${PROJECT_BINARY_DIR}/test/ctrl DESTINATION tool
-          PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE )
+#install ( FILES ${PROJECT_BINARY_DIR}/test/libtool.so DESTINATION tool )
+#install ( FILES ${PROJECT_BINARY_DIR}/test/ctrl DESTINATION tool
+#          PERMISSIONS OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE )
 
 ## Packaging directives
 set ( CPACK_GENERATOR "DEB" "RPM" "TGZ" )
