--- a/library/src/CMakeLists.txt	2019-08-13 20:22:27.795619540 +0200
+++ b/library/src/CMakeLists.txt	2019-08-13 20:24:33.162619061 +0200
@@ -90,6 +90,7 @@
   target_compile_options( rocfft PRIVATE -Wno-unused-command-line-argument )
 
   foreach( target ${AMDGPU_TARGETS} )
+    target_compile_options( rocfft PRIVATE --amdgpu-target=${target} )
     target_link_libraries( rocfft PRIVATE --amdgpu-target=${target} )
   endforeach( )
 else( CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" )
