--- a/library/src/rocsparse_module.f90	2020-08-30 10:47:18.323910928 +0200
+++ b/library/src/rocsparse_module.f90	2020-08-30 10:47:52.140910799 +0200
@@ -99,9 +99,9 @@
 
 !   rocsparse_layer_mode
     enum, bind(c)
-        enumerator :: rocsparse_layer_mode_none = x'0'
-        enumerator :: rocsparse_layer_mode_log_trace = x'1'
-        enumerator :: rocsparse_layer_mode_log_bench = x'2'
+        enumerator :: rocsparse_layer_mode_none = 0
+        enumerator :: rocsparse_layer_mode_log_trace = 1
+        enumerator :: rocsparse_layer_mode_log_bench = 2
     end enum
 
 !   rocsparse_status
