--- ROCm-OpenCL-Runtime-roc-2.7.0/api/opencl/amdocl/CMakeLists.old	2019-08-07 14:45:41.637970284 +0200
+++ ROCm-OpenCL-Runtime-roc-2.7.0/api/opencl/amdocl/CMakeLists.txt	2019-08-07 14:47:06.638967528 +0200
@@ -72,6 +72,7 @@
   $<TARGET_OBJECTS:oclrocm>
   ${ADDITIONAL_LIBRARIES}
 )
+target_link_directories(amdocl64 PUBLIC ${Clang_DIR}/lib)
 if(${USE_COMGR_LIBRARY} MATCHES "yes")
   target_link_libraries(amdocl64 oclelf pthread dl ${ROCT_LIBRARIES} ${ROCR_LIBRARIES})
 else()
