--- ROCm-OpenCL-Runtime-roc-2.6.0/runtime/CMakeLists.old	2019-07-22 16:14:04.613165639 +0200
+++ ROCm-OpenCL-Runtime-roc-2.6.0/runtime/CMakeLists.txt	2019-07-22 16:15:13.173163416 +0200
@@ -63,6 +63,8 @@
   ${CMAKE_SOURCE_DIR}/compiler/tools/caching/cache.cpp
 )
 set_target_properties(oclruntime PROPERTIES POSITION_INDEPENDENT_CODE ON)
+add_dependencies(oclruntime opencl1.2-c.amdgcn.inc_target)
+add_dependencies(oclruntime opencl2.0-c.amdgcn.inc_target)
 
 set(AMDGCN_DEP_LIST)
 if(AMDGCN_TARGETS_LIB_DEPS)
