--- ROCm-OpenCL-Runtime-roc-2.6.0/runtime/device/devprogram.old	2019-07-09 04:19:44.000000000 +0200
+++ ROCm-OpenCL-Runtime-roc-2.6.0/runtime/device/devprogram.cpp	2019-07-22 17:02:13.776071958 +0200
@@ -1552,7 +1552,7 @@
 }
 #else // not using COMgr
 bool Program::linkImplLC(amd::option::Options* options) {
-#if defined(WITH_LIGHTNING_COMPILER) || defined(USE_COMGR_LIBRARY)
+#if 0 && defined(WITH_LIGHTNING_COMPILER) || defined(USE_COMGR_LIBRARY)
   using namespace amd::opencl_driver;
   internal_ = (compileOptions_.find("-cl-internal-kernel") != std::string::npos) ?
     true : false;
