--- ROCm-OpenCL-Runtime-roc-2.6.0/CMakeLists.old	2019-07-22 15:24:16.547262527 +0200
+++ ROCm-OpenCL-Runtime-roc-2.6.0/CMakeLists.txt	2019-07-22 15:25:27.224260235 +0200
@@ -86,25 +86,6 @@
 
 add_subdirectory(tools/clinfo)
 
-install(PROGRAMS $<TARGET_FILE:clang> $<TARGET_FILE:lld>
-        DESTINATION bin/x86_64
-        COMPONENT libraries)
-
-install(PROGRAMS $<TARGET_FILE:OpenCL>
-        DESTINATION lib/x86_64
-        COMPONENT applications)
-
-install(PROGRAMS $<TARGET_LINKER_FILE:OpenCL>
-        DESTINATION lib/x86_64
-        COMPONENT libraries)
-
-install(DIRECTORY
-        "${CMAKE_CURRENT_SOURCE_DIR}/api/opencl/khronos/headers/opencl2.2/CL"
-        DESTINATION include
-        COMPONENT libraries
-        USE_SOURCE_PERMISSIONS
-        PATTERN cl_egl.h EXCLUDE)
-
 foreach(AMDGCN_LIB_TARGET ${AMDGCN_LIB_TARGETS})
   get_target_property(lib_file_name ${AMDGCN_LIB_TARGET} ARCHIVE_OUTPUT_NAME)
   get_target_property(lib_file_path ${AMDGCN_LIB_TARGET} ARCHIVE_OUTPUT_DIRECTORY)
