--- a/CMakeLists.txt	2019-01-23 23:16:03.269629799 +0100
+++ b/CMakeLists.txt	2019-01-23 23:17:12.313628728 +0100
@@ -41,6 +41,9 @@
 include_directories(${CMAKE_SOURCE_DIR}/usr/lib/llvm/roc-2.0.0/include/llvm/Target)
 include_directories(${CMAKE_BINARY_DIR}/usr/lib/llvm/roc-2.0.0/include/llvm/Target)

+link_directories(/usr/lib/llvm/roc-2.2.0/lib)
+include_directories(/usr/lib/llvm/roc-2.2.0/include/)
+
 add_subdirectory(compiler/driver)
 
 set(BUILD_HC_LIB OFF CACHE BOOL "")
