--- rocm-opencl-runtime-2.6.0/cmake/modules/FindROCT.old        2019-07-19 13:53:59.562676107 +0200
+++ rocm-opencl-runtime-2.6.0/cmake/modules/FindROCT.cmake      2019-07-19 13:58:48.534666737 +0200
@@ -4,8 +4,8 @@
 #   - ROCT_FOUND     - ROCT status (found or not found)
 #   - ROCT_INCLUDES  - Required ROCT include directories
 #   - ROCT_LIBRARIES - Required ROCT libraries
-find_path(FIND_ROCT_INCLUDES hsakmt.h HINTS /opt/rocm/include)
-find_library(FIND_ROCT_LIBRARIES hsakmt HINTS /opt/rocm/lib)
+find_path(FIND_ROCT_INCLUDES hsakmt.h HINTS /usr/include/libhsakmt)
+find_library(FIND_ROCT_LIBRARIES hsakmt HINTS /usr/lib64)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(ROCT DEFAULT_MSG
