--- a/rocm_bandwidth_test_report.cpp	2021-09-05 11:58:23.995786884 +0200
+++ b/rocm_bandwidth_test_report.cpp	2021-09-05 11:58:21.137786894 +0200
@@ -341,7 +341,7 @@
   double* perf_matrix = new double[agent_index_ * agent_index_]();
   PopulatePerfMatrix(peak, perf_matrix);
   PrintPerfMatrix(false, peak, perf_matrix);
-  free(perf_matrix);
+  delete[] perf_matrix;
 }
 
 void RocmBandwidthTest::DisplayValidationMatrix() const {
@@ -349,7 +349,7 @@
   double* perf_matrix = new double[agent_index_ * agent_index_]();
   PopulatePerfMatrix(true, perf_matrix);
   PrintPerfMatrix(true, true, perf_matrix);
-  free(perf_matrix);
+  delete[] perf_matrix;
 }
 
 void RocmBandwidthTest::DisplayDevInfo() const {
