--- a/cmake/Dependencies.cmake	2020-04-10 14:18:38.285638041 +0200
+++ b/cmake/Dependencies.cmake	2020-04-10 14:19:51.443637762 +0200
@@ -19,23 +19,23 @@
 endif()
 
 # rocPRIM (https://github.com/ROCmSoftwarePlatform/rocPRIM)
-if(NOT DEFINED ROCPRIM_ROOT)
-  message(STATUS "Downloading and building rocPRIM.")
-  set(ROCPRIM_ROOT ${CMAKE_CURRENT_BINARY_DIR}/rocPRIM CACHE PATH "")
-  download_project(
-    PROJ                rocPRIM
-    GIT_REPOSITORY      https://github.com/ROCmSoftwarePlatform/rocPRIM.git
-    GIT_TAG             5fa0c79fbb199f83c5202d9e218311a78335a1f4
-    INSTALL_DIR         ${ROCPRIM_ROOT}
-    CMAKE_ARGS          -DBUILD_TEST=OFF -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -DCMAKE_PREFIX_PATH=/opt/rocm
-    LOG_DOWNLOAD        TRUE
-    LOG_CONFIGURE       TRUE
-    LOG_BUILD           TRUE
-    LOG_INSTALL         TRUE
-    BUILD_PROJECT       TRUE
-    UPDATE_DISCONNECTED TRUE
-  )
-endif()
+#if(NOT DEFINED ROCPRIM_ROOT)
+#  message(STATUS "Downloading and building rocPRIM.")
+#  set(ROCPRIM_ROOT ${CMAKE_CURRENT_BINARY_DIR}/rocPRIM CACHE PATH "")
+#  download_project(
+#    PROJ                rocPRIM
+#    GIT_REPOSITORY      https://github.com/ROCmSoftwarePlatform/rocPRIM.git
+#    GIT_TAG             5fa0c79fbb199f83c5202d9e218311a78335a1f4
+#    INSTALL_DIR         ${ROCPRIM_ROOT}
+#    CMAKE_ARGS          -DBUILD_TEST=OFF -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -DCMAKE_PREFIX_PATH=/opt/rocm
+#    LOG_DOWNLOAD        TRUE
+#    LOG_CONFIGURE       TRUE
+#    LOG_BUILD           TRUE
+#    LOG_INSTALL         TRUE
+#    BUILD_PROJECT       TRUE
+#    UPDATE_DISCONNECTED TRUE
+#  )
+#endif()
 find_package(rocprim REQUIRED CONFIG PATHS ${ROCPRIM_ROOT})
 
 # Test dependencies
