diff -Naur '--exclude=rocprim' a/cmake/Dependencies.cmake b/cmake/Dependencies.cmake
--- a/cmake/Dependencies.cmake	2019-06-08 21:52:54.768818572 +0200
+++ b/cmake/Dependencies.cmake	2019-06-08 21:53:55.824818339 +0200
@@ -44,22 +44,23 @@
 endif()
 
 # rocPRIM (https://github.com/ROCmSoftwarePlatform/rocPRIM)
-message(STATUS "Downloading and building rocPRIM.")
-set(ROCPRIM_ROOT ${CMAKE_CURRENT_BINARY_DIR}/rocPRIM CACHE PATH "")
-download_project(
-  PROJ           rocPRIM
-  GIT_REPOSITORY https://github.com/ROCmSoftwarePlatform/rocPRIM.git
-  GIT_TAG        develop
-  INSTALL_DIR    ${ROCPRIM_ROOT}
-  CMAKE_ARGS     -DBUILD_TEST=OFF -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
-  LOG_DOWNLOAD   TRUE
-  LOG_CONFIGURE  TRUE
-  LOG_BUILD      TRUE
-  LOG_INSTALL    TRUE
-  BUILD_PROJECT  TRUE
-  ${UPDATE_DISCONNECTED_IF_AVAILABLE}
-)
-find_package(rocprim REQUIRED CONFIG PATHS ${ROCPRIM_ROOT})
+#message(STATUS "Downloading and building rocPRIM.")
+#set(ROCPRIM_ROOT ${CMAKE_CURRENT_BINARY_DIR}/rocPRIM CACHE PATH "")
+#download_project(
+#  PROJ           rocPRIM
+#  GIT_REPOSITORY https://github.com/ROCmSoftwarePlatform/rocPRIM.git
+#  GIT_TAG        develop
+#  INSTALL_DIR    ${ROCPRIM_ROOT}
+#  CMAKE_ARGS     -DBUILD_TEST=OFF -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
+#  LOG_DOWNLOAD   TRUE
+#  LOG_CONFIGURE  TRUE
+#  LOG_BUILD      TRUE
+#  LOG_INSTALL    TRUE
+#  BUILD_PROJECT  TRUE
+#  ${UPDATE_DISCONNECTED_IF_AVAILABLE}
+#)
+#find_package(rocprim REQUIRED CONFIG PATHS ${ROCPRIM_ROOT})
+find_package(rocprim REQUIRED CONFIG)
 
 # Test dependencies
 if(BUILD_TEST)
