--- a/rocm_smi/CMakeLists.txt	2020-11-15 09:44:48.915817077 +0100
+++ b/rocm_smi/CMakeLists.txt	2020-11-15 09:52:26.913815330 +0100
@@ -96,45 +96,39 @@
 endif ()
 
 ## Add symlinks from top level ROCm lib dir to rocm-smi lib so files
-if(${BUILD_SHARED_LIBS})
-  add_custom_target ( so-link ALL WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-                    COMMAND ${CMAKE_COMMAND} -E create_symlink
-                          ../${ROCM_SMI}/lib/${ROCM_SMI_LIB_NAME}.so so-link )
-  add_custom_target ( so-major-link ALL WORKING_DIRECTORY
-                  ${CMAKE_CURRENT_BINARY_DIR} COMMAND ${CMAKE_COMMAND}
-                  -E create_symlink
-                  ../${ROCM_SMI}/lib/${ROCM_SMI_LIB_NAME}.so.${VERSION_MAJOR}
-                                                               so-major-link )
-
-  install ( FILES ${CMAKE_CURRENT_BINARY_DIR}/so-link DESTINATION lib RENAME
-                                                     ${ROCM_SMI_LIB_NAME}.so )
-  install ( FILES ${CMAKE_CURRENT_BINARY_DIR}/so-major-link DESTINATION lib
-                             RENAME ${ROCM_SMI_LIB_NAME}.so.${VERSION_MAJOR} )
-endif()
+#if(${BUILD_SHARED_LIBS})
+#  add_custom_target ( so-link ALL WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+#                    COMMAND ${CMAKE_COMMAND} -E create_symlink
+#                          ../${ROCM_SMI}/lib/${ROCM_SMI_LIB_NAME}.so so-link )
+#  add_custom_target ( so-major-link ALL WORKING_DIRECTORY
+#                  ${CMAKE_CURRENT_BINARY_DIR} COMMAND ${CMAKE_COMMAND}
+#                  -E create_symlink
+#                  ../${ROCM_SMI}/lib/${ROCM_SMI_LIB_NAME}.so.${VERSION_MAJOR}
+#                                                               so-major-link )
+#
+#  install ( FILES ${CMAKE_CURRENT_BINARY_DIR}/so-link DESTINATION lib RENAME
+#                                                     ${ROCM_SMI_LIB_NAME}.so )
+#  install ( FILES ${CMAKE_CURRENT_BINARY_DIR}/so-major-link DESTINATION lib
+#                             RENAME ${ROCM_SMI_LIB_NAME}.so.${VERSION_MAJOR} )
+#endif()
 ## Add the install directives for the runtime library.
 install(TARGETS ${ROCM_SMI_TARGET}
-          LIBRARY DESTINATION ${ROCM_SMI}/lib
-          ARCHIVE DESTINATION ${ROCM_SMI}/lib
+          LIBRARY DESTINATION lib64
+          ARCHIVE DESTINATION lib64
           COMPONENT ${ROCM_SMI_COMPONENT})
 
-install(FILES ${COMMON_SRC_ROOT}/include/rocm_smi/rocm_smi.h
-                                        DESTINATION rocm_smi/include/rocm_smi)
-install(FILES ${COMMON_SRC_ROOT}/include/rocm_smi/kfd_ioctl.h
-                                        DESTINATION rocm_smi/include/rocm_smi)
-install(FILES ${COMMON_SRC_ROOT}/python_smi_tools/rsmiBindings.py
-                                        DESTINATION rocm_smi/bindings)
-install(FILES ${COMMON_SRC_ROOT}/python_smi_tools/rocm_smi.py
-       PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
-                         GROUP_WRITE GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-                                        DESTINATION bin)
-install(FILES ${COMMON_SRC_ROOT}/python_smi_tools/rocm-smi
-                                        DESTINATION bin)
-add_custom_target(bindings_link ALL
-                    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-                    COMMAND ${CMAKE_COMMAND} -E create_symlink
-                       ../rocm_smi/bindings/rsmiBindings.py bindings_link)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/bindings_link
-                     DESTINATION bin RENAME rsmiBindings.py)
+install(FILES ${COMMON_SRC_ROOT}/include/rocm_smi/rocm_smi.h DESTINATION include/rocm_smi)
+install(FILES ${COMMON_SRC_ROOT}/include/rocm_smi/kfd_ioctl.h DESTINATION include/rocm_smi)
+#install(FILES ${COMMON_SRC_ROOT}/python_smi_tools/rsmiBindings.py DESTINATION bin)
+#install(FILES ${COMMON_SRC_ROOT}/python_smi_tools/rocm_smi.py PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
+#                         GROUP_WRITE GROUP_EXECUTE WORLD_READ WORLD_EXECUTE DESTINATION bin)
+#install(FILES ${COMMON_SRC_ROOT}/python_smi_tools/rocm-smi DESTINATION bin)
+#add_custom_target(bindings_link ALL
+#                   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+#                    COMMAND ${CMAKE_COMMAND} -E create_symlink
+#                       ../rocm_smi/bindings/rsmiBindings.py bindings_link)
+#install(FILES ${CMAKE_CURRENT_BINARY_DIR}/bindings_link
+#                     DESTINATION bin RENAME rsmiBindings.py)
 
 
 # Generate Doxygen documentation
