--- a/bin/hipcc
+++ b/bin/hipcc
@@ -841,7 +841,7 @@
         $toolArgs .= " -Wl,--enable-new-dtags -Wl,--rpath=$HIP_LIB_PATH:$ROCM_PATH/lib -lhip_hcc ";
       }
       # To support __fp16 and _Float16, explicitly link with compiler-rt
-      $toolArgs .= " -L$HIP_CLANG_PATH/../lib/clang/$HIP_CLANG_VERSION/lib/linux -lclang_rt.builtins-x86_64 "
+#      $toolArgs .= " -L$HIP_CLANG_PATH/../lib/clang/$HIP_CLANG_VERSION/lib/linux -lclang_rt.builtins-x86_64 "
     }
 }
 
--- a/hip-config.cmake.in
+++ b/hip-config.cmake.in
@@ -234,18 +234,18 @@
   endif()
 
   # Add support for __fp16 and _Float16, explicitly link with compiler-rt
-  set_property(TARGET hip::host APPEND PROPERTY
-    INTERFACE_COMPILE_OPTIONS -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
-  )
-  set_property(TARGET hip::host APPEND PROPERTY
-    INTERFACE_LINK_LIBRARIES -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
-  )
-  set_property(TARGET hip::device APPEND PROPERTY
-    INTERFACE_COMPILE_OPTIONS -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
-  )
-  set_property(TARGET hip::device APPEND PROPERTY
-    INTERFACE_LINK_LIBRARIES -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
-  )
+#  set_property(TARGET hip::host APPEND PROPERTY
+#    INTERFACE_COMPILE_OPTIONS -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
+#  )
+#  set_property(TARGET hip::host APPEND PROPERTY
+#    INTERFACE_LINK_LIBRARIES -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
+#  )
+#  set_property(TARGET hip::device APPEND PROPERTY
+#    INTERFACE_COMPILE_OPTIONS -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
+#  )
+#  set_property(TARGET hip::device APPEND PROPERTY
+#    INTERFACE_LINK_LIBRARIES -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64
+#  )
 endif()
 
 set( hip_LIBRARIES hip::host hip::device)
