--- a/bin/hipcc	2019-09-02 13:51:52.464620538 +0200
+++ b/bin/hipcc	2019-09-02 13:54:22.451410499 +0200
@@ -273,7 +273,7 @@
     $HIPCXXFLAGS .= " -isystem $HSA_PATH/include";
     $HIPCXXFLAGS .= " -Wno-deprecated-register";
 
-    $HIPLDFLAGS .= " -L$HSA_PATH/lib -L$ROCM_PATH/lib -lhsa-runtime64 -lhc_am ";
+    $HIPLDFLAGS .= " -L$HSA_PATH/lib64 -L$ROCM_PATH/lib64 -lhsa-runtime64 -lhc_am ";
 #    $HIPLDFLAGS .= " -L$HCC_HOME/compiler/lib -lLLVMAMDGPUDesc -lLLVMAMDGPUUtils -lLLVMMC -lLLVMCore -lLLVMSupport ";
 
     # Add trace marker library:
